### PR TITLE
Task/migration tool

### PR DESCRIPTION
**Pull request by dmoranj**
Implements #388

### CODE REVIEW

File: test/unit/examples/mongoCollections/configurations.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  {
+    "apikey" : "hs4h2ke7oerfdvrexbyh54w2",
+    "cbroker" : "http://10.0.0.2:1026",
+    "entity_type" : "device",
+    "resource" : "/iot/d",
+    "service" : "demo",
+    "service_path" : "/"
+  }
+]
```
Comment (gtorodelvalle) [inline test/unit/examples/mongoCollections/configurations.json]: Final blank line appreciated... :)

File: test/unit/examples/mongoCollections/devices.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        "object_id" : "a",
+        "name" : "AddSoap",
+        "type" : "Quantity",
+        "value" : "Initial value"
+      }
+    ],
+    "service" : "smart_gondor",
+    "service_path" : "/waters"
+  }
+]
```
Comment (gtorodelvalle) [inline test/unit/examples/mongoCollections/devices.json]: Final blank line appreciated... ;)

File: test/unit/examples/mongoCollections/configurations.json (hunk 1 above)
Comment (AlvaroVega) [inline test/unit/examples/mongoCollections/configurations.json]: wrong EOF character

**Comment by AlvaroVega** [comment]
LGTM

**Comment by gtorodelvalle** [comment]
LGTM

**Comment by XavierVal** [comment]
LGTM

**Pull request merged**